tests/security/pam/pam_config.pm

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: PAM tests for pam-config, create, add or delete service
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#70345, tc#1167579
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use registration qw(add_suseconnect_product remove_suseconnect_product);
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Install the pam-config package, since this service package is not installed by default
+    zypper_call 'in pam-config';
```

Comment:
```# rpm -qf /usr/sbin/pam-config
pam-config-1.1-1.32.x86_64
```
In `pam_basic_function.pm` command `pam-config` was invoked why not move this "installation" into this file?

Reply:
Good point,  I will modify the code

Reply:
done